chainweb.cabal

```diff
@@ -351,7 +351,7 @@ library
         , monad-control >= 1.0
         , mtl >= 2.2
         , mwc-random >= 0.13
-        , mwc-probability >= 2.0 && <2.2
+        , mwc-probability >= 2.0 && <2.3
```

Comment:
Is this upper bound needed at all? I am not aware of any issue with that package.

Reply:
It was just me lifting the restriction, I'm not sure that there was any API-related issue.